clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/services/Front50Service.groovy

```diff
@@ -62,4 +61,13 @@ interface Front50Service {
 
   @DELETE('/v2/tags/{id}')
   Response deleteEntityTags(@Path('id') String id)
+
+  // v2 MPT APIs
+  @GET('/v2/pipelineTemplates/{pipelineTemplateId}')
+  Map getV2PipelineTemplate(@Path("pipelineTemplateId") String pipelineTemplateId,
```

Comment:
Does `listIds` support `version` and `digest` as parameters?  It looks like it just accepts `scopes`

Reply:
No, just scopes.